content/docs/command-reference/root.md

```diff
@@ -29,22 +30,78 @@ refer to a file in another directory.
 
 ```dvc
 $ dvc root
-
 .
 
 $ cd subdir
 $ dvc root
-
 ..
 ```
 
 ## Example: Referencing files
 
+Simplify file references when working in a subdirectory of a <abbr>DVC
+project</abbr>.
+
 ```dvc
 $ dvc root
-
 ../../../
 
 $ dvc run -d $(dvc root)/data/file.cvs ... \
     python $(dvc root)/scripts/something.py
 ```
+
+## Example: Output references
+
+Simplify output file or directory references.
+
+```dvc
+$ dvc root
+../../../
+
+$ dvc get -o $(dvc root)/root-model.pkl \
+    https://github.com/iterative/example-get-started model.pkl
+```
```

Comment:
Thanks but seems to me like the same use case as the previous example, and probably `run` is much more realistic. Doubt people will need this for `get`.

Reply:
> > I adjusted the second example... uses dvc root to specify the output of a dvc get command...
> 
> I don't see this in the 2nd example.

Sorry, I meant to refer to this third example. 

I agree the use cases are similar. I used `get` because it doesn't rely on how `dvc run` executes or how stages are generated. As someone new to dvc, this seemed more straightforward to understanding how `dvc root` works on its own.

I think that examples should get progressively more complex but was hesitant to move or replace the existing second example on the first pass.

Reply:
Yes, its good thining. But tbh `dvc root` is far from a well-known or frequently-used command so we don't need so many examples. The minimum possible is best here.

Reply:
Removed this example.

Reply:
Thanks